NodeFirefox/Dockerfile

```diff
@@ -1,22 +1,24 @@
-FROM selenium/node-base:2.53.0
+FROM selenium/node-base:2.53.1
 MAINTAINER Selenium <selenium-developers@googlegroups.com>
 
 USER root
 
 #=========
 # Firefox
 #=========
-ENV FIREFOX_VERSION 45.0.2
 RUN apt-get update -qqy \
   && apt-get -qqy --no-install-recommends install firefox \
   && rm -rf /var/lib/apt/lists/* \
-  && wget --no-verbose -O /tmp/firefox.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
-  && apt-get -y purge firefox \
-  && rm -rf /opt/firefox \
-  && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
-  && rm /tmp/firefox.tar.bz2 \
-  && mv /opt/firefox /opt/firefox-$FIREFOX_VERSION \
-  && ln -fs /opt/firefox-$FIREFOX_VERSION/firefox /usr/bin/firefox
+  && apt-get -y purge firefox
+RUN wget --no-verbose -O /tmp/firefox-45.0.2.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/45.0.2/linux-x86_64/en-US/firefox-45.0.2.tar.bz2 \
```

Comment:
I agree that a for loop makes sense here, and if I get a chance to work on this I'll update it. I'm out of the office now for a week, and then traveling for 10 days, so I'm not sure how much time I'll have to work on it. I think the environment variable suggestion is too restrictive as we'll need a way of doing pre-releases too. Perhaps a variable which is a list of versions `FIREFOX_VERSIONS=45.0.2 46.0.1 47.0.1` which can then be used in the for loop?